Configurations.md

```diff
@@ -1248,12 +1248,20 @@ Control the case of the letters in hexadecimal literal values
 
 ## `hide_parse_errors`
 
-Do not show parse errors if the parser failed to parse files.
+This option is deprecated. Use `show_parse_errors` instead. 
```

Comment:
Final ask. Let's provide some explanation in the docs for why we're renaming this option. Something like:

This option is deprecated and has been renamed to `show_parse_errors` to avoid confusion around the double negative default of `hide_parse_errors=false`.